src/SDL/Video.hs

```diff
@@ -184,8 +187,9 @@ data WindowConfig = WindowConfig
   , windowOpenGL       :: Maybe OpenGLConfig -- ^ Defaults to 'Nothing'. Can not be changed after window creation.
   , windowPosition     :: WindowPosition     -- ^ Defaults to 'Wherever'.
   , windowResizable    :: Bool               -- ^ Defaults to 'False'. Whether the window can be resized by the user. It is still possible to programatically change the size by changing 'windowSize'.
-  , windowInitialSize  :: V2 CInt            -- ^ Defaults to @(800, 600)@. If you set 'windowHighDPI' flag, window size in screen coordinates may differ from the size in pixels. Use 'glGetDrawableSize' to get size in pixels.
+  , windowInitialSize  :: V2 CInt            -- ^ Defaults to @(800, 600)@. If you set 'windowHighDPI' flag, window size in screen coordinates may differ from the size in pixels. Use 'glGetDrawableSize' or 'SDL.Video.Vulkan.vkGetDrawableSize' to get size in pixels.
   , windowVisible      :: Bool               -- ^ Defaults to 'True'.
+  , windowVulkan       :: Bool               -- ^ Defaults to 'False'. If 'True', function 'SDL.Video.Vulkan.vkLoadLibrary' 'Nothing' will be called automatically before first window creation, and 'SDL.Video.Vulkan.vkUnloadLibrary' will be called after last window destruction.
```

Comment:
My only concern here is that you can set `windowpenGL` *and* `windowVulkan` on a window - but does that make sense?

Reply:
You are right! SDL [returns error is this case](https://github.com/spurious/SDL-mirror/blob/563e2c2d71e39e07eeb23ee141f7ce6fe9319543/src/video/SDL_video.c#L1463). I think we should reflect this in our docs. Are you agree?

Of course, we can do something like
```haskell
data WindowVideoConfig = NoVideo | OpenglWindow OpenGLConfig | VulkanWindow
```
but it will be tangible breaking change.

Reply:
It's actually already a breaking change because we've changed the type of `Window`. I do think some kind of enumeration is the way to go here.

Reply:
Ok, I'll try such an algebraic type in the near future.